src/commands/fun/flipCoin.ts

```diff
@@ -0,0 +1,22 @@
+import { Message } from 'discord.js';
+import { ApplyOptions } from '@sapphire/decorators';
+import { Command, CommandOptions } from '@sapphire/framework';
+import { BOT_PREFIX } from '../../bot';
+
+@ApplyOptions<CommandOptions>({
+  aliases: ['fc', 'flip', 'flip-coin', 'coin-flip', 'coinflip'],
+  description: 'Flip a coin! In making decisions, if it is not great, at least it is fair!',
+  detailedDescription: `**Example:** \`${BOT_PREFIX}flip-coin\`\n
```

Comment:
"Example" should be plural and there should be a `\n` between it and the first example. I guess this is just to follow Commando's format of displaying examples in the help command.
![image](https://user-images.githubusercontent.com/58180935/157529657-77d1f5ab-f82f-45a4-882e-a9127174450d.png)